dev-requirements.txt

```diff
@@ -3,7 +3,8 @@ codecov
 doctr
 ipython
 matplotlib
+pymongo
 sphinx
 sphinx_rtd_theme
-psdm_qs_cli
+git+https://github.com/slaclab/psdm_qs_cli.git
```

Comment:
What does this do? If it checks it out from git when you load dev-requirements then this isn't good practice.

Reply:
Actually this completely stalled the travis on closer inspection